notebooks/population_to_catalog.ipynb

```diff
@@ -0,0 +1,445 @@
+{
```

Comment:
<p>this cell seems a bit odd</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>